src/components/structures/FilePanel.js

```diff
@@ -50,6 +50,10 @@ class FilePanel extends React.Component {
         if (room?.roomId !== this.props?.roomId) return;
         if (toStartOfTimeline || !data || !data.liveEvent || ev.isRedacted()) return;
 
+        if (ev.shouldAttemptDecryption()) {
+            ev.attemptDecryption(room._client._crypto);
```

Comment:
we shouldn't be accessing private members like this, there's no guarantee they won't change between releases